docs/ui/interactions/fragments/web/chatbot.md

```diff
@@ -0,0 +1,11 @@
+A simple way to add a conversational UI into your app is to use our ChatBot Component.
+
+ChatBot automatically renders a complete chat messaging interface that can be used out-of-the-box, or it can be customized using theming support.
+
+<inline-fragment src=~/ui/interactions/fragments/web/installation.md></inline-fragment>
+
+<inline-fragment src="~/ui/interactions/fragments/web/usage.md"></inline-fragment>
```

Comment:
These fragments seem pretty chatbot specific, can we put them in a sub folder? `web/chatbot/usage.md`

Reply:
Sure